kaldb/src/main/java/com/slack/kaldb/server/OpenSearchBulkIngestApi.java

```diff
@@ -241,6 +241,7 @@ private KafkaProducer<String, byte[]> createKafkaTransactionProducer(String tran
     props.put("key.serializer", "org.apache.kafka.common.serialization.StringSerializer");
     props.put("value.serializer", "org.apache.kafka.common.serialization.ByteArraySerializer");
     props.put("transactional.id", transactionId);
+    props.put("linger.ms", 250);
```

Comment:
As discussed, we'll followup with supporting `KAFKA_ADDITIONAL_PROPS` here as well in some format as we probably want to support `snappy`, etc.